rllib/execution/parallel_requests.py

```diff
@@ -371,15 +371,18 @@ def add_workers(self, new_workers: Union[List[ActorHandle], ActorHandle]) -> Non
             if new_worker not in self._all_workers:
                 self._all_workers.append(new_worker)
 
-    def remove_workers(self, workers: Union[List[ActorHandle], ActorHandle]) -> None:
+    def remove_workers(
+        self,
+        workers: Union[List[ActorHandle], ActorHandle],
+        remove_in_flight_requests: bool = False,
```

Comment:
Just to learn about how we use this: In what case would we want to remove a worker without removing in-flight requests?

Reply:
Maybe if the worker got "reassigned" to some other worker pool?
Not sure either. So far, we only use this method for when workers failed, so it makes sense here to also erase their in-flight requests tracking (otherwise, these in-flight requests would fail repeatedly and could lead to getting stuck in an endless loop).

Reply:
why not make the default True if there isn't a case where we shouldn't remove the in-flight requests?